spec/features/admin/order_cycles/complex_editing_multiple_product_pages_spec.rb

```diff
@@ -34,7 +34,10 @@
     end
 
     scenario "select all products" do
-      check "order_cycle_incoming_exchange_0_select_all_variants"
+      # replace with scroll_to method when upgrading to Capybara >= 3.13.0
+      cb_id = "order_cycle_incoming_exchange_0_select_all_variants"
```

Comment:
`cb_id` is a bit cryptic. I would call it `checkbox_id`.

Reply:
Good point - changed to be more informative